### PR TITLE
feat: provide build instructions for launchpad

### PR DESCRIPTION
**Pull request by CjS77**
..and deprecate docker-rig instructions

### CODE REVIEW

File: applications/launchpad/README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+1. Rust and cargo (https://www.rust-lang.org/tools/install).
+2. NodeJs (v16.0 or higher) and Yarn (v 1.22 or higher).
+3. Tauri CLI (`cargo install tauri-cli`). _Optional_.
+4. [Docker](https://docs.docker.com/get-docker/) is not _strictly_ a pre-requisite, since the launchpad on-boarding 
+   flow will install it for you, but you will need docker eventually, so putting it here.
+5. Install the front-end dependencies
+   ```text
+   $ cd applications/launchpad  
+   $ yarn
+   $ cd gui-react
```
Comment (leet4tari) [inline applications/launchpad/README.md:16]: Needs to be changed to gui-vue

Comment (CjS77) [reply applications/launchpad/README.md:16]: No it doesn't :)

`gui-vue` is deprecated and will be removed. Also the person who wrote it should be barred form touching front-end code ever again

Comment (leet4tari) [reply applications/launchpad/README.md:16]: cool, I was on an old branch

File: applications/launchpad/README.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+a blockchain data, you can use something like this to extract it to the host filesystem:
+
+`docker run --rm -v $(pwd):/backup -v blockchain:/blockchain ubuntu tar czvf /backup/backup.tar.gz /blockchain`
+
+
+## Layout
+
+      +-----------------------+
+      |                       |
++---->|    Console Wallet     +------------------+
```
Comment (leet4tari) [inline applications/launchpad/README.md]: Need four spaces before any ascii art, I think ... Would be nice to quote, but I have not found a way.

File: applications/launchpad/README.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      |                       |
+      +-----------------------+
+
+## Building custom docker images
+
+The docker images for the base node, wallet etc. are designed to handle the broadest set of chipsets and 
+architectures. For this reason, they not be optimal for _your_ system. You can build custom images for launchpad 
+using the `build_images.sh` script in this folder.
+
+You can also set the following environment variables that are passed to the rust compiler:
```
Comment (leet4tari) [inline applications/launchpad/README.md]: Need to review the info on envs and notes for them.

There are three samples current. And you can't really build/target none native architecture/platform.

**Comment by leet4tari** [comment]
LGTM

**Review by tomaszantas — APPROVED** [review]

**Pull request merged**